crates/vm/src/system/public_values/core.rs

```diff
@@ -164,7 +164,9 @@ where
         &mut self,
         state: VmStateMut<TracingMemory, CTX>,
         instruction: &Instruction<F>,
-        row_slice: &mut [F],
+        trace: &mut [F],
```

Comment:
i want to switch this to abstract buffer later

Reply:
cool, we can modify later